src/utils/constants.js

```diff
@@ -45,7 +45,8 @@ export const NUMBER_FRACTION_THRESHOLD = 100000
 export const MAXIMUM_FRACTION_DIGITS = 20
 export const APP_TITLE_SHORT = 'æScan'
 export const APP_TITLE = 'æScan - æternity Blockchain Explorer'
-export const APP_DESCRIPTION = 'æScan is a blockchain explorer, analytics platform, and decentralized Smart Contract navigation platform based on æternity'
+export const APP_DESCRIPTION = 'æScan is a blockchain explorer, analytics platform, and decentralized Smart Contract navigation platform based on æternity.'
+export const APP_KEYWORDS = 'aescan, aeternity, blockchain, explorer, search, crypto, currency'
```

Comment:
In my opinion I would remove "search, crypto, currency".

Reply:
I took that inspiration from etherscan and it seems to be working pretty well for them. @janmichek WDYT?
![image](https://github.com/aeternity/aescan/assets/46789227/c53d3800-7a01-4a13-a8ea-27ff96e4bd59)

Reply:
My understanding of this topic is low, but the motivation behind choosing such keywords make complete sense to me

Reply:
Since we would be left with only 4 keywords which are well within the usual number of keywords, I'll keep them then.